Dockerfile

```diff
@@ -22,7 +22,7 @@ COPY requirements.in .
 # add the code as the final step so that when we modify the code
 # we don't bust the cached layers holding the dependencies and
 # system packages.
-COPY scispacy/ scispacy
+COPY scispacy/ scispacy/
```

Comment:
I think maybe `COPY SciSpaCy/ scispacy/` would work? or renaming the github repo and SciSpaCy folder. I'm not sure why it would work locally but not in teamcity though

Reply:
But this is copying the python package inside the git repo, not the actual repo right? Or is the build step run such that it copies the entire repo?

Reply:
Ok something weird is happening, it didn't pick up my changes to the package...